### DISCUSSION TITLE
Industrial TNT can destroy claim protected area

### DISCUSSION THREAD

**Discussion by anoldshoe** [GriefPrevention/GriefPrevention#1105] [Q&A] [answered]
While other explosives block damage is disabled and does not effect claim protection, Industrial TNT will destroy blocks.

Tested on a public server, where one person claimed an area, and I activated the ITNT in, and then around, the claim.

Explosives were disabled at the time, and is by default. regular TNT did not damage blocks in the same claim.

GT New Horrizons server
MC 1.7.10

**Comment by RoboMWM** [comment]
the hecc is industrial tnt?

**Comment by RoboMWM** [comment]
If it's a mod, then please see the note about mods (wherever it was, probably in documentation? I'll have to dig it out I guess) - they need to fire bukkit events else there's 0 way for GP to know about 'em

**Reply by anoldshoe** [reply]
Please explain "fire bukkit events"? The modpack we are using is called GT New Horizons. Thanks in advance

**Reply by bigpresh** [reply]
GriefPrevention works by intercepting standard Bukkit API events, for instance `EntityExplodeEvent`, `EntityChangeBlockEvent` etc, deciding whether it should be prevented, and if so, cancelling the event.

If third party mods are not firing those events, then GriefPrevention cannot cancel them if they're in a claim etc because it will never know about them.

**Comment by Jikoo** [comment, marked as answer]
Industrial TNT is from a mod, yeah.

Unfortunately, if the server mod you're running doesn't fire events, there's nothing GP can do about it. Consider disabling the recipe in your CraftTweaker config (assuming that was even available back in 1.7.10).
